build.gradle

```diff
@@ -28,5 +27,18 @@ dependencies {
     compile "org.apache.httpcomponents:httpcore:4.2.4"
     compile "org.apache.httpcomponents:httpmime:4.3"
     compile 'com.google.zxing:core:3.2.1'
+}
 
+sourceSets {
+    main.java.srcDir "src/main"
```

Comment:
as our project has the standard project layout this was not necessary here , because by default gradle assumes Production Java source is at src/main/java . 
reference - https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_project_layout
Otherwise we can set the source dir if we had different project layout .